tools/make/image.mk

```diff
@@ -67,6 +66,32 @@ image.push.%:
 	@echo "===========> Pushing docker image tag $(IMAGE):$(TAG) for $(ARCH)"; \
 	$(DOCKER) push $(IMAGE):$(TAG); \
 
+.PHONY: image.multiarch.verify
+image.multiarch.verify:
+	$(eval pass := $(shell ))
+	@docker buildx --help | grep -qw "docker buildx" || { \
+		echo "Cannot find `docker buildx`, please install first"; \
+		exit 1; }
+
+.PHONY: image.multiarch.emulate $(EMULATE_TARGETS)
+image.multiarch.emulate: $(EMULATE_TARGETS)
+$(EMULATE_TARGETS): image.multiarch.emulate.%:
+	@docker run --rm --privileged tonistiigi/binfmt --install linux/$* # Install QEMU emulator, the same emulator as the host will report an error but can safe ignore
+
+.PHONY: image.multiarch.setup
+image.multiarch.setup: image.verify image.multiarch.verify image.multiarch.emulate
+	@docker run --rm --privileged tonistiigi/binfmt --install all # Install QEMU emulators
```

Comment:
shouldnt this line be deleted @daixiang0 ?

Reply:
I miss it, post anther PR to fix it.